packages/mdc-textfield/README.md

```diff
@@ -340,19 +350,26 @@ complicated.
+| addClassToBottomLine(className: string) => void | Adds a class to the bottom line element |
+| removeClassFromBottomLine(className: string) => void | Removes a class from the bottom line element |
 | addClassToHelptext(className: string) => void | Adds a class to the help text element. Note that in our code we check for whether or not we have a help text element and if we don't, we simply return. |
 | removeClassFromHelptext(className: string) => void | Removes a class from the help text element. |
 | helptextHasClass(className: string) => boolean | Returns whether or not the help text element contains the current class |
-| setHelptextAttr(name: string, value: string) => void | Sets an attribute on the help text element |
-| removeHelptextAttr(name: string) => void | Removes an attribute on the help text element |
 | registerInputFocusHandler(handler: EventListener) => void | Registers an event listener on the native input element for a "focus" event |
 | deregisterInputFocusHandler(handler: EventListener) => void | Un-registers an event listener on the native input element for a "focus" event |
 | registerInputBlurHandler(handler: EventListener) => void | Registers an event listener on the native input element for a "blur" event |
 | deregisterInputBlurHandler(handler: EventListener) => void | Un-registers an event listener on the native input element for a "blur" event |
 | registerInputInputHandler(handler: EventListener) => void | Registers an event listener on the native input element for an "input" event |
 | deregisterInputInputHandler(handler: EventListener) => void | Un-registers an event listener on the native input element for an "input" event |
 | registerInputKeydownHandler(handler: EventListener) => void | Registers an event listener on the native input element for a "keydown" event |
-| deregisterInputKeydownHandler(handler: EventListener) => void | Un-registers an event listener on the native input element for a "keydown" event |
+| deregisterInputKeydownHandler(handler: EventListener) => void | Deregisters an event listener on the native input element for a "keydown" event |
+| registerInputPointerDownHandler(handler: EventListener) => void | Registers an event listener on the native input element for a pointer event |
+| deregisterInputPointerDownHandler(handler: EventListener) => void | Deregisters an event listener on the native input element for a "pointer" event |
+| registerTransitionEndHandler(handler: EventListener) => void | Registers an event listener on the native input element for a "transitionend" event |
+| deregisterTransitionEndHandler(handler: EventListener) => void | Deregisters an event listener on the native input element for a "transitionend" event |
+| setBottomLineAttr(attr: string, value: string) => void | Sets an attribute with a given value on the bottom line element |
+| setHelptextAttr(name: string, value: string) => void | Sets an attribute on the help text element |
```

Comment:
To be consistent with the previous line, "Sets an attribute with a given value on the help text element"